### PR TITLE
New extension cloudservice

### PR DESCRIPTION
**Pull request by qwordy**
https://github.com/Azure/azure-rest-api-specs/blob/master/specification/compute/resource-manager/Microsoft.Compute/preview/2020-10-01-preview/cloudService.json

---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

**Review by yungezz — COMMENTED** [review]
what's the gen.zip file?

**Comment by qwordy** [comment]
> 
> 
> what's the gen.zip file?

A magical file that stores test modification information. It is used to prevent overwriting modified test when generating again.

File: src/cloudservice/azext_cloudservice/action.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+# pylint: disable=wildcard-import
+# pylint: disable=unused-wildcard-import
+
+from .generated.action import *  # noqa: F403
```
Comment (yungezz) [inline src/cloudservice/azext_cloudservice/action.py:13]: why those files not under azext_cloudservice/generated folder?

File: src/cloudservice/azext_cloudservice/manual/_help.py [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+# pylint: disable=too-many-lines
+
+from knack.help_files import helps
+
+
+helps['cloud-service role-instance'] = """
```
Comment (yungezz) [inline src/cloudservice/azext_cloudservice/manual/_help.py:15]: hi @qwordy , this RP has lots of manual codes, could you pls share why it need manual customization? then we can share that which part isn't covered by codegen yet.

Comment (qwordy) [reply src/cloudservice/azext_cloudservice/manual/_help.py:15]: I've created issues on https://github.com/Azure/autorest.az/issues/created_by/qwordy
image

File: src/cloudservice/azext_cloudservice/action.py (hunk 1 above)
Comment (qwordy) [reply src/cloudservice/azext_cloudservice/action.py:13]: We have three `action.py`. It is magic from autorest. This one is existing one. Others are "generated" one and "manual" one.

**Review by yungezz — APPROVED** [review]

**Pull request merged**